src/Widgets/DataSearchWidget/DataSearchWidgetEditingConfig.ts

```diff
@@ -17,7 +17,11 @@ provideEditingConfig(DataSearchWidget, {
         { value: 'btn-secondary', title: 'Secondary color' },
       ],
     },
-    placeholder: { title: 'Placeholder text' },
+    placeholder: {
+      title: 'Placeholder text',
+      description:
+        'This value is also utilized for the aria label of the search button.',
```

Comment:
* [ ] FYI ARIA is uppercase (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)